scripts/01_deploy_nft_marketplace.ts

```diff
@@ -18,7 +18,7 @@ const deployMarketplace: DeployFunction<Settings> = async (
   const result = await deployer.deployContract(NFTMarketPlace, {
     // @ts-ignore
     initialFields: initialFields,
-    gasAmount: 100000
+    gasAmount: 50000
```

Comment:
Why the gas amount cann't be estimated well automatically?

Reply:
I will investigate and try to fix this in a seperate PR, WDYT?

Reply:
This seems to be the [cause](https://github.com/alephium/alephium-nft/pull/7#discussion_r1199699453), I am working on a fix.

Reply:
Proposed [fix](https://github.com/alephium/dev-alephium/pull/862)

Reply:
Fixed.